### DISCUSSION TITLE
Consider adding details and replacements for deprecated comments

### DISCUSSION THREAD

**Discussion by leungyauming** [v2fly/v2ray-core#667] [General]
Some items inside v2ray's codebase are deprecated for years but the comment of those items are not informational.

For example, in [app/features/inbound/inbound.go](https://github.com/v2fly/v2ray-core/blob/master/features/inbound/inbound.go) file, the GetRandomInboundProxy function of the Handler struct is deprecated, but the details and the replacements for that functions is not stated, adding difficulties to development.

Please consider adding "Use xxx.xxx instead.' section to the comment of the deprecated items.

**Comment by Loyalsoldier** [comment]
You can continue working on the PR https://github.com/v2fly/v2ray-core/pull/553 of mine if you like to.

**Reply by leungyauming** [reply]
Ok I will take a look first.

**Reply by Loyalsoldier** [reply]
Any deprecated code should be pushed to branch `v5`.

**Reply by leungyauming** [reply]
What do you mean deprecated code should be pushed to branch `v5`?
Wasn't deprecated code expected to be removed/replaced?

**Reply by Loyalsoldier** [reply]
Any deprecated code should be removed from branch `v5`.

**Reply by leungyauming** [reply]
ok understand
